plugins/experimental/slice/slice.h

```diff
@@ -32,8 +32,6 @@
 #define PLUGIN_NAME "slice"
 #endif
 
-constexpr std::string_view X_CRR_IMS_HEADER = {"X-Crr-Ims"};
```

Comment:
This constant and all references to `X_CRR_IMS_HEADER` were removed, but `SLICER_MIME_FIELD_INFO` still exists in `HttpHeader.h` (line 39). Consequently there are two areas that still reference the old default `X-Slicer-Info` header name using the constant, and it is injected into requests despite having a "custom" header configuration.